Add key generation procedures (#19)
openpgp-pqc/draft-openpgp-pqc: draft-wussler-openpgp-pqc.md

An implementation MUST support SHA3-256 and SHOULD support SHA3-512, in
order to support the hash binding with Dilithium + ECC signatures.

### Key generation procedure {#ecc-dilithium-generation}

The implementation MUST independently generate the Dilithium and the ECC

> component keys. The dilithium component is generated as specified in

Reviewer: ```suggestion
component keys. The Dilithium component is generated as specified in
```

Reviewer: Typo

> component keys. The dilithium component is generated as specified in

Reviewer: ```suggestion
component keys. Dilithium key generation follows the specification in
```

Reviewer: Aligned to the text for Kyber, fixed typo.

An implementation supporting a specific SPHINCS+ algorithm and parameter MUST
also support the matching hash algorithm.

### Key generation

> To generate a SPHINCS+ key the function `spx_keygen()` specified in

Reviewer: ```suggestion
The SPHINCS+ key generation is performed according to the function `spx_keygen()` specified in
```

Reviewer: There was something missing like "the function is used" in the previous sentence.
